src/com/dotcms/api/system/event/verifier/PermissionVerifier.java

```diff
@@ -0,0 +1,34 @@
+package com.dotcms.api.system.event.verifier;
+
+
+import com.dotcms.api.system.event.Payload;
+import com.dotcms.api.system.event.PayloadVerifier;
+import com.dotcms.exception.BaseRuntimeInternationalizationException;
+import com.dotcms.rest.api.v1.system.websocket.SessionWrapper;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.PermissionAPI;
+import com.dotmarketing.business.Permissionable;
+import com.dotmarketing.exception.DotDataException;
+
+/**
+ * Created by freddyrodriguez on 16/9/16.
+ */
+public class PermissionVerifier implements PayloadVerifier<Integer>{
+
+    private PermissionAPI permissionAPI = APILocator.getPermissionAPI();
```

Comment:
This should be final, and should be pass by the constructor follow the approach you follow on other classes